<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSessionWidget</class>
 <widget class="QWidget" name="VideoSessionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoSessionWidget</string>
  </property>
  <property name="styleSheet">
   <string>
QWidget#MainFrame {
	background-color: rgb(255, 255, 255);
	color: rgb(0,0,0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="mainVerticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QFrame" name="infoBarFrame">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QLabel" name="connectionStatusTitle">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string>color: rgb(0,0,0);</string>
                 </property>
                 <property name="text">
                  <string>Connection:</string>
                 </property>
                 <property name="indent">
                  <number>6</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="connectionStatus">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string>color: rgb(0,0,0);</string>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="audioStatusTitle">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string>color: rgb(0,0,0);</string>
                 </property>
                 <property name="text">
                  <string>Audio:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="audioStatus">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string>color: rgb(0,0,0);</string>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="videoStatusTitle">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string>color: rgb(0,0,0);</string>
                 </property>
                 <property name="text">
                  <string>Video:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="videoStatus">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string>color: rgb(0,0,0);</string>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>1</width>
                   <height>1</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="closePushButton">
                 <property name="styleSheet">
                  <string>QPushButton {
	font-size: 10px;
	color: rgb(0,0,0);
	padding: 0px;
	padding-right: 3px;
	padding-left: 3px;
	min-height: 20px;
	max-height: 20px;
	border: 0px;
	border-color: rgba(0, 0, 0, 0);
	border-top: 1px solid rgb(69, 159, 255);
	border-left: 1px solid rgb(69, 159, 255);
	border-radius: 0px;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover {
	font-size: 10px;
	color: rgb(0,0,0);
	padding: 0px;
	padding-right: 3px;
	padding-left: 3px;
	min-height: 20px;
	max-height: 20px;
	border: 0px;
	border-color: rgba(0, 0, 0, 0);
	border-top: 1px solid rgb(69, 159, 255);
	border-left: 1px solid rgb(69, 159, 255);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:0.006, y1:0, x2:0, y2:1, stop:0.142045 rgba(229, 234, 243, 255), stop:0.295455 rgba(209, 215, 221, 255), stop:0.744318 rgba(178, 183, 196, 255), stop:1 rgba(222, 227, 236, 255));
}

QPushButton::pressed {
	color: rgb(0,0,0);
	padding-right: 3px;
	padding-left: 3px;
	min-height: 20px;
	max-height: 20px;
	border: 0px;
	border-color: rgba(0, 0, 0, 0);
	border-top: 1px solid rgb(69, 159, 255);
	border-left: 1px solid rgb(69, 159, 255);
	border-radius: 0px;
	background-color: rgb(207, 207, 207);
}</string>
                 </property>
                 <property name="text">
                  <string>Close</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
